### DISCUSSION TITLE
como cobrar um freela/(serviço) que não te pagam?

### DISCUSSION THREAD

**Discussion by rubenmarcus** [frontendbr/forum#1128] [Discussão]
Vocês já passaram pela situação de fazer um trabalho, muitas vezes por fora, perdendo horas de sono, e o cliente, muitas vezes um ex cliente, ou conhecido/amigo, some e não te pagar??

O que vocês fazem ?

**Comment by VZiemer** [comment]
Já sofri muito com isso, principalmente com aquela "correçãozinha fácil que não vai te tomar tempo nenhum", minha sugestão, por experiência é que independente da relação pessoal estipule um valor e peça parte adiantado (50%), penso que seja um ato de boa fé e confiança de ambas as partes.

**Comment by woliveiras** [comment]
Já tive esse tipo de treta.

Você fez contrato, mano?

**Comment by kivervinicius** [comment]
Sempre cobro meu custo antecipado, se pelo menos alguma coisa ocorrer errado, vc já supriu seus custos antecipadamente, restando somente o lucro para receber após, contrato é importante para essas situações pois caso ocorrer você pode protestar.

**Comment by rubenmarcus** [comment]
Então , esse é meu erro 90% dos meus freelas fiz sem contrato, porque era com pessoas que 'confiava', ou com quem ja trabalhei, e me pagaram certinho, mesmo sem contrato.

O problema é exatamente esse, a maioria dos clientes, não quer pagar antecipado, como também quer ficar fazendo modificações o tempo todo, ou que você carregue eles no colo, e depois de implementado o sistema, você faça tudo por eles.

No meu caso a maioria dos freelas é em wordpress, mas recentemente fiz um em Moodle, que o cliente tinha suporte de 1 ano, e passado 1 ano, eles vieram protestar e falaram que iam processar porque não usaram a manutenção , e agora precisava usar.

O cliente em questão queria que ficasse, criando e manejando o conteúdo por ele, e a pessoa que me contratou pra isso (um ex-chefe), pediu em carater de urgência, mas na hora de pagar não respondeu mais.

Eu passei por isso no ínicio dos meus freelas, quando fui numa agência de marketing, e o cara fechou 1 site bem grande comigo, e fechou o escopo, mas não detalhou mais nada do projeto e sumiu foi viajar, me devendo um dinheirão, e depois apareceu pedindo tudo em 1 final de semana.

**Comment by rubenmarcus** [comment]
@kivervinicius  Cara tenho idéia, que isso é pra profissionais já senior ,ou renomado no mercado. Porque no mercado brasileiro pelo menos eu vejo a maioria de freelas, sites de freela, e tb freelas por fora, a galera quer um leilão, e nem só isso, o cliente parece que quer que vc pegue ele no colo, afague, faça cafuné, e ainda bata palma e agradeça pra ele.

**Comment by rodrigooler** [comment]
É sempre bom cobrar um custo inicial do projeto pra arcar com os gastos inicias, e também contrato bem feito, pois só assim você conseguirá cobrar conforme o combinado. E lembrando que deve sempre documentar por e-mail as conversas, valores etc.

Se o cara não te pagar mete ele na justiça e recebe tudo aquilo que foi combinado e que tu tenha direito ;).

**Comment by fakenickels** [comment]
**Sempre cobrar um valor inicial**. Se o valor for baixo, cobrar 50%. Se o valor já for mais alto - seilá, 3k pra cima - cobrar 20-30% adiantado e **não começar** até que a quantia seja paga.

**Comment by rodrigooler** [comment]
Sim essa prática é muito boa porque te previne daqueles clientes que repassa o job para mais de uma pessoa e escolhe quem pagar no final... geralmente são os espertinhos que passam a perna.

**Comment by kivervinicius** [comment]
@rubenmarcus eu tenho adotado muito a entrada no cartão de crédito (divido até em 4x) sempre coloco embutido o custo do financiamento do cartão, tem dado muitoo certo, por que na hora que vc fala para um cliente que vc precisa de 2mil de entrada ou mais eles assustam pois geralmente não tem para o investimento inicial, dai quando vc fala q divide, eles fecham em 95% dos casos, se chorar faço todo o projeto em 10x.

**Comment by rubenmarcus** [comment]
sim ,mas no meu caso era um suporte pós projeto, e estava sendo cotado por hora. Com esse cliente, sempre tinha listas / planilhas o que foi feito por hora, e ai o que seria pago.

Suporte tipo , eu sei que fiz errado na primeira lista de suporte, eu deveria ter cobrado já parte das horas, o cliente que agiu de ma fé

**Comment by kivervinicius** [comment]
entendi, bom quando for estes casos estabelece horas mínimas de projeto, e já cobra uma porcentagem em cima delas, infelizmente vamos pensar mil maneiras de evitar mas sempre teremos dor de cabeça com confiança excessiva e clientes agindo de má fé.

**Comment by rodrigooler** [comment]
Talvez tenha alguma startup que resolva esse problema @rubenmarcus se não tiver... seria um ótimo tema. Como assegurar que o freelancer ou pessoa que trabalhe fora da CLT receba o dinheiro sem dor de cabeça. É um problema coletivo, tanto que tem inúmeros profissionais que sofrem com esse problema no dia a dia.

**Comment by luizbills** [comment]
Por essas e outras que só trabalho com pagamento adiantado. Mas se você gosta de receber depois, faça contrato.

**Comment by rubenmarcus** [comment]
@rodrigooler  alguns serviços e plataformas como Toptal e Upwork , garantem isso. Mas já ouvi casos de outras plataformas como 99 Freelas e Workana, em que a plataforma sempre deu caso de ganho pro cliente.

**Comment by rodrigooler** [comment]
@rubenmarcus então, no caso o ideal seria utilizar plataformas que garantem o teu recebimento, ou pedir uma consultoria com algum advogado pra ver seus direitos como prestador de serviço para elaborar um contrato que beneficie ambas as partes e não prejudique ninguém. Pois já aconteceu comigo de entregar job e o cara não me pagar, e eu não ter nada alem e emails e código para comprovar, até me alertaram antes de fazer um contrato para ele assinar... mas vacilei na época.

**Comment by woliveiras** [comment]
Galera, tirando essa parte da proteção antes do começo do trampo (pagamento antecipado e o contrato), como vocês fazem pra **cobrar** de alguém que não lhes paga pelo job?

Eu costumava tratar tudo por emails, pegar muitos dados da pessoa com a desculpa de manter o contato e, se essa pessoa não me pagasse, eu tinha assim os meios pra encontrar ela.

Se ela atrasa ou mesmo some, eu tentava o contato por diversas vezes e deixava tudo registrado. Se, mesmo com diversas tentativas de contato ela não me respondesse, então eu recorreria a justiça. É abrir um B.O. pela prestação de serviços mesmo e deixar a brasa comer solta. :joy: 

Abrir um B.O. nunca foi necessário comigo, pois eu sempre fiz freela para pessoas mais próximas e quando comecei a fazer pra gente desconhecida eu saquei de um contrato pra botar pressão. Quando elas não pagaram, esse contato várias vezes já resolvia. Eu fazia igual empresa de cobrança mesmo :hocho: :dollar:.

Dá muito trabalho? Daaaaa

Dá mais dor de cabeça ainda? Daaaaa

Mas a gente recebe o nosso dinheiro e, dependendo do transtorno, você mete um danos morais no infeliz que nos passou a perna.

**Comment by giovannism20** [comment]
Vocês têm um contrato que podemos chamar de padrão ou mesmo para tomar como base para freelancers ? @woliveiras @rubenmarcus @rodrigooler @luizbills @kivervinicius @grsabreu @VZiemer
